### PR TITLE
partial Revert "feat(hack): make command use correct creds"

### PR DESCRIPTION
**Pull request by RiRa12621**
Reverts openshift/route-monitor-operator#30

And adds build support via dockerized build

### CODE REVIEW

**Comment by georgettica** [comment]
Just unrevert the app-sre-build-deploy section.
It was my bad then.

And it won't work 100% because of this container running as root.

And the fact that kustomize is not dockerfiled

File: hack/pipeline.dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN microdnf install -y golang make which
+RUN microdnf install -y git
+
+ARG BIN=operator-sdk
+COPY $BIN /usr/local/bin/operator-sdk
```
Comment (georgettica) [inline hack/pipeline.dockerfile:7]: default binary path doesn't work, and it won't work on the remote slave

File: hack/app_sre_create_image_catalog.sh [hunk 2]
```diff
@@ -9,8 +9,11 @@ _OPERATOR_NAME="route-monitor-operator"
 BRANCH_CHANNEL="$1"
 QUAY_IMAGE="$2"
 
+# Build an image locally that has all tools we need
+docker build -f hack/pipeline.dockerfile -t pipelinebuilder:latest ./hack/
+
 # generate the bundle folder
-make bundle
+make bundle OPERATOR_SDK_COMMAND="docker run --rm -w /root/tmp -i -v $(pwd):/root/tmp pipelinebuilder:latest"
```
Comment (georgettica) [inline hack/app_sre_create_image_catalog.sh:16]: we would also need to override the binary to kustomize with `KUSTOMIZE=`

Comment (RiRa12621) [reply hack/app_sre_create_image_catalog.sh:16]: yes you're right

**Comment by RiRa12621** [comment]
/lgtm
🤞

**Review by NautiluX — COMMENTED** [review]
/lgtm

**Comment by georgettica** [comment]
@NautiluX it seems like I have the power!

**Pull request merged**